haproxy.cfg

```diff
@@ -32,29 +32,22 @@ frontend http_front
     acl host_equitytool req.hdr(host) -i -m beg equitytool.austinmobility.io
     redirect scheme https code 301 if host_equitytool
 
-    # PostgREST Path ACLs
-    acl path_knack_services path_beg /knack-services
-    acl path_legacy_scripts path_beg /legacy-scripts
-    acl path_ctr_data_lake path_beg /ctr-data-lake
-    acl path_parking path_beg /parking
-    acl path_road_conditions path_beg /road-conditions
-    acl path_bond_reporting path_beg /bond-reporting
-
-    # Host and source ACLs
+    # Only go to postgrest if the path is valid AND:
+    #   - either source is 10.0.0.0/8
+    #   - or source is 162.89.100.0/24 and host header is present
+
+    acl path_postgrest path_beg /knack-services /legacy-scripts /ctr-data-lake /parking /road-conditions /bond-reporting
     acl host_postgrest hdr(host) -i atd-postgrest.austinmobility.io
     acl src_postgrest_internal src 10.0.0.0/8
     acl src_postgrest_coa_network src 162.89.100.0/24
 
-    # Only go to postgrest if the path is valid AND:
-    #   - either source is 10.0.0.0/8
-    #   - or source is 162.89.100.0/24 and host header is present
-    use_backend postgrest if (path_knack_services or path_legacy_scripts or path_ctr_data_lake or path_parking or path_road_conditions or path_bond_reporting) and ( src_postgrest_internal or ( src_postgrest_coa_network and host_postgrest ) )
+    use_backend postgrest if path_postgrest src_postgrest_internal
+    use_backend postgrest if path_postgrest src_postgrest_coa_network host_postgrest
 
     use_backend server_not_found
 
 frontend https_front
-    bind *:443 ssl crt /usr/local/etc/haproxy/ssl alpn h2,http/1.1
-
+    bind *:443 ssl crt /usr/local/etc/haproxy/ssl/equitytool.austinmobility.io.pem alpn h2,http/1.1
```

Comment:
Here’s a clearer version of that paragraph:
We include a README.md in this folder to ensure Git keeps the directory, so others know it’s where certs should go. However, `haproxy` complains because README.md is not a valid PEM file. Since HAProxy requires strict syntax compliance to start, this change switches from loading certs using a directory glob to loading each PEM file explicitly.

See https://github.com/cityofaustin/atd-data-tech/issues/22212.

Reply:
Tangential, but I guess we need to remember to renew this cert and install it here? I'm wondering what's the best way to remember to do this. Or is this one on certbot?

Reply:
just saw this comment and wanted to link to the issue for the auto certbot renewal: [#22212](https://github.com/cityofaustin/atd-data-tech/issues/22212) - Automate certbot SSL/TLS certificate renewal for EAZ tool and enhance compose / haproxy usage

@frankhereford i had a related suggestion in your EAZ PR for a link or mention in those docs so that we know that the cert (and future renewal) are handled in this repo https://github.com/cityofaustin/dts-equity-analysis-zones/pull/8#pullrequestreview-2795397628